### PR TITLE
[wings] Avoid double setting visibility in Wings converter

### PR DESCRIPTION
**Pull request by no-reply**
`#visibility` in `Hydra::AccessControls` is a projection on the ACLs. Since the
converter sets the ACL users and groups manually, there's no need to set
`visibility` separately; doing so may complicate special cases like embargo and
lease.

This is a non-functional change, isolated to Wings.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: lib/wings/active_fedora_converter.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def initialize(resource:)
     end
 
     ##
-    # Accesses and parses the attributes from the resource through ConverterValueMapper
+    # Accesses and parses the attributes from the resource through ConvetrerValueMapper
```
Comment (jcoyne) [inline lib/wings/active_fedora_converter.rb]: Typo?

**Comment by revgum** [comment]
fixes #3806

File: lib/wings/active_fedora_converter.rb (hunk 1 above)
Comment (no-reply) [reply lib/wings/active_fedora_converter.rb]: i suppose so, but not one introduced in this PR.

**Comment by no-reply** [comment]
@revgum rebased.

File: lib/wings/active_fedora_converter.rb (hunk 1 above)
Comment (jcoyne) [reply lib/wings/active_fedora_converter.rb]: why is GH showing it as a change then?

Comment (no-reply) [reply lib/wings/active_fedora_converter.rb]: oh huh. i assumed it was a whitespace change, but i guess i must have done a stealth letter transpose in my editor.

sorry for the confusion, and thanks for the catch!

**Comment by jrgriffiniii** [comment]
The failing test appears to be unrelated (I've encountered this on my branches: https://circleci.com/gh/samvera/hyrax/4287#tests/containers/2), I'll restart the build.

**Pull request merged**